### DISCUSSION TITLE
Support for PEP 665

### DISCUSSION THREAD

**Discussion by ThatXliner** [python-poetry/poetry#4931] [Ideas]
Ok, before I say anything, I should say that **I know almost nothing about this**.

The only reason I knew about PEP 665 (which is, correct me if I'm wrong, a proposal for a standard lock file format) was because I saw [Frostming's new repo](https://github.com/frostming/pep665_poc):

<img width="647" alt="Screen Shot 2021-12-22 at 4 24 10 PM" src="https://user-images.githubusercontent.com/66848002/147169631-d18c788b-fe49-4654-883d-af8de7f8ae5e.png">

And then I got curious so I googled "PEP 665" and found [this](https://www.python.org/dev/peps/pep-0665/)

So what do you guys think?

**Comment by clintonroy** [comment]
There's a lot of communication between the tool and pep authors, look at the poetry section of https://www.python.org/dev/peps/pep-0665/#potential-tool-support

**Reply by ThatXliner** [reply]
Like I said, I've never actually thoroughly read the PEP. I guess that's what I get for just skimming the first several parts in the table of contents.

Well, thanks for the answer!
